### PR TITLE
add con2net with logo and extension

### PR DESCRIPTION
**Pull request by con2net**

### CODE REVIEW

**Comment by xchs** [comment]
Two things:
* Could you please rename your SVG logo to `logo.svg`?
* Could you please optimize your SVG logo with e.g. [SVGMO](https://jakearchibald.github.io/svgomg/)?

File: meta/con2net/contao-hibp-bundle/en.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+en:
+    title: HIBP "Have I Been Pwned" - Check E-Mail-Address
+    description: >
+        This Extension allows you to check an e-mail address against the databases of "Have I Been Pwned" (https://haveibeenpwned.com)
```
Comment (xchs) [inline meta/con2net/contao-hibp-bundle/en.yml:4]:
```suggestion
        This extension allows you to check an e-mail address against the databases of "Have I Been Pwned" (https://haveibeenpwned.com)
```

**Comment by con2net** [comment]
Tx, xchs!

**Comment by xchs** [comment]
Thank you @con2net

**Pull request merged**